### DISCUSSION TITLE
My Character Doesnt Play Its Singing Anims

### DISCUSSION THREAD

**Discussion by BassicalyYT** [YoshiCrafter29/YoshiCrafterEngine#356] [Q&A]
I ported my character to yoshi engine and I did everything it told me to do and it just wont work please help

https://user-images.githubusercontent.com/98070641/210139196-faf1ad4a-89ea-4150-9229-925441e1fee4.mp4

![picture](https://user-images.githubusercontent.com/98070641/210139232-a848e4e0-26a7-4a7d-a87d-0e05a4d91245.PNG)

**Comment by TheKitBoi** [comment]
> I ported my character to yoshi engine and I did everything it told me to do and it just wont work please help

there supposed to be 'sing' on anim and a capitals after sing that it has to do for example
```
idle
singLEFT
singDOWN
singUP
singRIGHT
```
